start-finalSetup02Modpack

```diff
@@ -78,18 +78,26 @@ done
 fi
 
 if [[ "$MANIFEST" ]]; then
-EFFECTIVE_MANIFEST_URL=$(curl -Ls -o /dev/null -w %{url_effective} $MANIFEST)
-case "X$EFFECTIVE_MANIFEST_URL" in
+    if [[ -e "$MANIFEST" ]]; then
+        EFFECTIVE_MANIFEST_FILE=$MANIFEST
+    elif isURL "$MANIFEST"; then
+        EFFECTIVE_MANIFEST_FILE=/tmp/manifest.json
+        curl -Ls -o $EFFECTIVE_MANIFEST_FILE "$MANIFEST"
```

Comment:
A previous contribution (https://github.com/itzg/docker-minecraft-server/commit/df9909a8e4c499f5d6b9c0bc327a7bba6ea52a9a) discovered that the weird-looking-two-step process was needed to resolve the effective URL and then download the file from that.

I think this combination of the previous line and yours would do it:
```suggestion
        EFFECTIVE_MANIFEST_URL=$(curl -Ls -o /dev/null -w %{url_effective} $MANIFEST)
        curl -Ls -o $EFFECTIVE_MANIFEST_FILE "$EFFECTIVE_MANIFEST_URL"
```

Reply:
Made these changes and pushed in new commit